pisa/stages/abs/pi_earth_absorption.py

```diff
@@ -0,0 +1,244 @@
+"""
+PISA pi stage for the calculation of earth layers and survival probabilities.
+
+The stage calculates first the depth of a water column that is mass-equivalent
+to the path traversed by the neutrino through the earth. This is done
+using the same Layers module that is also used for oscillation.
+The survival probability is then calculated from the average cross-section
+with protons and neutrons.
+"""
+from __future__ import absolute_import, print_function, division
+
+import numpy as np
+import ROOT
```

Comment:
Since this needs ROOT, we should make sure it is declared as an optional dependency in the INSTALL.md / setup.py

Reply:
I found the place in INSTALL.md where ROOT is mentioned as an optional dependency, but I didn't find anything in setup.py. What do you mean that it should also be listed there?

Reply:
that's probably fine if it is not in setup.py, as ROOT might be not pip installable. In INSTALL.md it currently says:
"ROOT >= 6.12.04 with PyROOT Necessary for xsec.genie and unfold.roounfold services, and to read ROOT cross section files in the crossSections utils module. Due to a bug in ROOT's python support (documented here https://github.com/IceCubeOpenSource/pisa/issues/430), you need at least version 6.12.04 of ROOT." So there it should be mentionned that is also needed for your stage, and if you need a specific version etc

Reply:
Okay, I moved the import to the setup function and mention the stage in INSTALL.md in the line you quote.